### PR TITLE
Gateway

### PR DESCRIPTION
**Pull request by R2wenD2**
cc: @vbatts

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ jobs:
 
       - run:
           name: Install go dependencies
-          command: go get -u -v github.com/golang/protobuf/protoc-gen-go
+          command: go get -u -v github.com/golang/protobuf/protoc-gen-go;
```
Comment (skelterjohn) [inline .circleci/config.yml]: list the multiple packages as multiple arguments to the same 'go get' command.

Comment (R2wenD2) [reply .circleci/config.yml]: done

**Comment by R2wenD2** [comment]
Add grpc-gateway support, next up I need to implement the main package to use the gateway.

**Comment by vbatts** [comment]
oh cool! Now to ensure folks can easily read the RESTful API translation.

**Comment by R2wenD2** [comment]
First I need to get the gateway service running :) that's a WIP.

**Comment by vbatts** [comment]
OIC :grimacing:

**Pull request merged**